ui/raidboss/data/05-shb/raid/e5s.js

```diff
@@ -6,25 +6,31 @@
     ko: /^희망의 낙원 에덴: 공명편\(영웅\) \(1\)$/,
   },
   timelineFile: 'e5s.txt',
+  timelineTriggers: [
+    {
+      id: 'E5S Stepped Leader Next',
+      regex: /^Stepped Leader$/,
+      beforeSeconds: 15,
+      run: function(data) {
+        data.steppedLeaderNext = true;
+      },
+    },
+  ],
   triggers: [
     {
-      id: 'E5S Fury\'s Bolt Gain',
-      regex: Regexes.gainsEffect({ target: 'Ramuh', effect: 'Fury\'s Bolt', capture: false }),
-      regexDe: Regexes.gainsEffect({ target: 'Ramuh', effect: 'Wütender Blitz', capture: false }),
-      regexFr: Regexes.gainsEffect({ target: 'Ramuh', effect: 'Boules de foudre', capture: false }),
-      regexJa: Regexes.gainsEffect({ target: 'ラムウ', effect: 'チャージボルト', capture: false }),
+      id: 'E5S Surge Protection Gain',
+      regex: Regexes.gainsEffect({ effect: 'Surge Protection' }),
+      condition: Conditions.targetIsYou(),
       run: function(data) {
-        data.fury = true;
+        data.surgeProtection = true;
       },
     },
     {
-      id: 'E5S Fury\'s Bolt Lose',
-      regex: Regexes.losesEffect({ target: 'Ramuh', effect: 'Fury\'s Bolt', capture: false }),
-      regexDe: Regexes.losesEffect({ target: 'Ramuh', effect: 'Wütender Blitz', capture: false }),
-      regexFr: Regexes.losesEffect({ target: 'Ramuh', effect: 'Boules de foudre', capture: false }),
-      regexJa: Regexes.losesEffect({ target: 'ラムウ', effect: 'チャージボルト', capture: false }),
+      id: 'E5S Surge Protection Loss',
+      regex: Regexes.losesEffect({ effect: 'Surge Protection' }),
+      condition: Conditions.targetIsYou(),
       run: function(data) {
-        data.fury = false;
+        data.surgeProtection = false;
```

Comment:
Do these effect messages only show up when you go from 0 to 1? Is there any message for more than that? Also half wondering if pro level triggers should look at times to figure out if this will run out before the mechanic you need it for too...

Reply:
The gain messages show up on every orb pickup.
The loss messages show up every time you lose the entirety of the buff or if you overcharge.

Reply:
Ah interesting, so if we wanted to track timers, you could just reset the timer on pickup.